android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -636,6 +636,28 @@ class XMTPModule : Module() {
             runBlocking { group?.sync() }
         }
 
+        AsyncFunction("addGroupMembers") { clientAddress: String, id: String, peerAddresses: List<String> ->
+            logV("addGroupMembers")
+            val client = clients[clientAddress] ?: throw XMTPException("No client")
+            if (client.libXMTPClient == null) {
+                throw XMTPException("Create client with enableAlphaMLS true in order to create a group")
+            }
+            val group = findGroup(clientAddress, id)
+
+            runBlocking { group?.addMembers(peerAddresses) }
```

Comment:
In the long-term it might be good to return an error if the group is not found, here and in the other methods, but doesn't need to block this PR